docs/source/task_tutorial.rst

```diff
@@ -254,7 +254,229 @@ And we have finished building our task.
 Task from Scratch
 ~~~~~~~~~~~~~~~~~
 
-Coming soon.
+In this section we will demonstrate the process of creating a task from scratch by adding the VQAv2 visual question-answering task. To implement this task we will inherit directly from the base ``Teacher`` class instead of using ``DialogTeacher``. This is usually not necessary, but it is done here as an example of creating a task from scratch.
+
+In this task, the agent is presented with an image of a scene and then asked to answer a question about that scene. A sample episode is demonstrated below. 
+
+::
+
+    [vqa_v2]: What is this man holding?
+    3rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrsssAssss9
+    s::::,:,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,::::::::G
+    A:;;;;;;;;;;;;:;;;;;;;;;;;;;;;;;;;;;;::::;;;;;;;;G
+    A;cccccc;;;;;;;;;;;;cccccc;;;;;ccc;;;;;;ccccccccc#
+    A;ccccccr32rccccccccccrrccccccccccccccccrrccccccc#
+    A;;;;ccc#@@3ccccrrrrrrrcrrccccccrrrrrrrrccccccccc#
+    A;c;;;;r@@@h;cccccccccccrrrrrrrrrcccccccccccccrrc#
+    A;c;cc;c8@&Hccccccccccccccccccccccccccccccccccccc#
+    A;cccccc3&@Hccrcccccccccccccccccrcccccccccccccccc#
+    Accccccc3@9srccccccccccccccrrccccccccccccccccccrc#
+    2ccrrAGs;s:292ccccccrrrrrrrrccccccccccccccrrrrrrr#
+    3rrcs9@A...9@Grcccccccccccccccccrrrccccrrrrrrrrrr#
+    2ccA9@&s:::&@9rccccccccccccrrrrrrrrrrrrrrrrssssrr#
+    2cs8&h2s;;;#8@Acccccccrrrrrrrrrrrrrrrrrsrrrrrssrr#
+    2rG@hrrrccr2h&hcrrrrrrrrrrrrrrrrrrrsrrrrrrrssssrr#
+    2r3@GrrrrrsAA9#rrrrrrrrrrrrrrrrrrrrrrrrssssssssss#
+    2rrh&GrrssAArG&srrrrrrrrrrrrsrssrssssssssssssrrrr#
+    2rrc28#A;crrrh@rcrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr#
+    2rrrcs9h,:;;c2&scrcrrrrrrrrrrrrccrrrrrrrrrrrrrrrr#
+    2cccccsr::;crs8Arrrrrrrrrrsrrrrrrrrrrrrrrrrrrrrrr#
+    2crrrrccc;cccr8Hrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrcc#
+    2cccccccsA2As388srrrrrrrrrrrrrrrrrrrrrrrrrrrccccc#
+    2rrrrrrh33HH#@#9srrrrrrrrrrrrrrrrrrrsssrrrrrrrrrr#
+    2rrrrrr#@&&&@@##hrrrrrrrrrrrrrrrrrrrrrrrsrsrrrsss#
+    3ssssssH@@@@@@&h#hArrrrrrrrrrrrsssrrrrrrrrrrrsssr#
+    3ssssssA&@88@@@H9hsrrrrrsssrrrrrrrrrssrssssssssss#
+    3sssssrs8@#2&@@##srsssssssssrssssssssssssssssssss#
+    3rrrrrrrG@#rH@@992rsssssssssssssssssssssssssssssr#
+    2rrrrrrrH@#sA9@893ssssssssssssssrrrrssssssssrrrrr#
+    2rrsssssH@GssH&&#hrsrrrrrrrrrrrrrrrrrrrrrrrssssss#
+    3rrsssssH@#srH&&GHrrrrrrrrrrrrrrsrrrrrrrrrssAAAAs9
+    3rssssssh@9srH@&GHrrrrsrssssssssssssssssssssAAAss9
+    3ssssssr2&&Arh@&G#hArrssssssssssssssssssssssAAAAA9
+    3ssssssrA8@Hrh&&G#H2sAssssssssssssssssssssAsAAAAs#
+    3ssssssrs#@&s3&@GG2AAAAAAssssssssssssssssssssssss#
+    3ssssssssh@@h28@#G2AAAAAsssssssssssssAsAAssssssss#
+    3ssssAAAAh&@Gh&@8#3AAAAsssssssssssssssssssAAAAAss9
+    3sssAAA22h&@GG@8##hAAssssssssssssssssssssA22AA2AA9
+    3sssssAAA2hh23h2AAAssssssssssssssssssssssAAAAAAAs9
+    H3333333333333333333333333333333333333333333333338
+
+    [labels: skateboard]
+       [Agent]: skateboard
```

Comment:
should I do that? the image would look nice definitely, but I like the ascii renderings hahah